RepairsListener/RepairsListener.csproj

```diff
@@ -19,8 +19,9 @@
     <PackageReference Include="Amazon.Lambda.Serialization.SystemTextJson" Version="2.1.0" />
     <PackageReference Include="Amazon.Lambda.SQSEvents" Version="2.0.0" />
     <PackageReference Include="AWSXRayRecorder.Handlers.AwsSdk" Version="2.8.3" />
-    <PackageReference Include="Hackney.Core.DynamoDb" Version="1.51.0" />
+    <PackageReference Include="Hackney.Core.DynamoDb" Version="1.79.0" />
```

Comment:
This can be removed - there's no dynamo integration

Reply:
![image](https://github.com/LBHackney-IT/repairs-listener/assets/70756861/c28dbc50-b7dd-4fb7-8601-5704bf1c2175)
I can remove all of these too, right?